conf.d/downloads

```diff
@@ -1,12 +1,13 @@
+
```

Comment:
The shebang needs to be on the first line; please remove this empty line.

Reply:
I went through and removed the extra empty line.